pkg/coremodel/playlist/testdata/valid/empty.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@sdboyer Why does Cue validate this file even after I set required fields? 🤔

Reply:
That's...surprising

There are some subtleties to the approach we take to validation, as indicated by [my comments in the `Validate()` implementation](https://github.com/grafana/thema/blob/ebeee841c10458674ab760308ca31657512dc999/impl.go#L336-L352). The fact that an empty struct makes it through... 😬 

If you add just _one_ of the now-required fields, does it start erroring as expected?